pkg/minishift/network/settings_windows.go

```diff
@@ -34,9 +33,9 @@ const (
 	networkingMessageName = "PROVISION_NETWORKING"
 )
 
-func ConfigureNetworking(machineName string, networkSettings NetworkSettings) {
+func ConfigureNetworking(machineName string, vmDriver string, networkSettings NetworkSettings) {
 	// Instruct the user that this does not work for other Hypervisors on Windows
-	if viper.GetString("vm-driver") != "hyperv" {
+	if vmDriver != "hyperv" {
```

Comment:
👍

Reply:
I am not a huge fan of string-based checks. Especially as all the follow-up code has to check this to decide on hypervisor specific solutions. Will address this in #1457